apps/indexers/nft-marketplace/src/model/generated/account.model.ts

```diff
@@ -0,0 +1,15 @@
+import {Entity as Entity_, Column as Column_, PrimaryColumn as PrimaryColumn_, OneToMany as OneToMany_} from "typeorm"
+import {Token} from "./token.model"
+
+@Entity_()
```

Comment:
What does the reason to use `Entity_()` instead of `Entity()` ?

Reply:
This is an auto generated code. `_` is a hack to avoid name collisions.